server/models/Request.js

```diff
@@ -31,10 +31,15 @@ const requestSchema = new Schema({
     enum: ["waiting", "matched", "referred"],
     required: true,
   },
-  scale: {
-    type: Number,
+  created_date: {
+    type: Date,
     required: true,
-  }
+    default: Date.now,
+  },
```

Comment:
thay vì làm thế này, m có thể add timestamp ở cuối model kiểu như vầy: 
<img width="585" alt="Screenshot 2024-04-03 at 9 35 18 AM" src="https://github.com/hoangr1010/Cupid/assets/117782458/8993ef4e-835b-457d-8600-8e0178b1bd73">
Ref: https://mongoosejs.com/docs/timestamps.html#:~:text=Mongoose%20schemas%20support%20a%20timestamps,this%20document%20was%20last%20updated